src/app.js

```
@@ -1 +1,76 @@
+/* eslint-disable no-console */
 'use strict';
+
+import fs from 'fs';
+import path from 'path';
+
+function copyFile() {
+  if (process.argv.slice(2).length !== 2) {
+    console.error('length is less');
+
+    return;
+  }
+
+  const [entryFile, copyTo] = process.argv.slice(2);
+
+  if (!entryFile || !copyTo) {
+    console.error('One or two param is undefined');
+
+    return;
+  }
+
+  let realSource;
+  let realDest;
+
+  try {
+    realSource = fs.realpathSync(entryFile);
+  } catch {
+    console.error('Source file does not exist');
+
+    return;
+  }
+
+  try {
+    realDest = fs.realpathSync(copyTo);
+  } catch {
+    realDest = path.resolve(copyTo);
+  }
+
+  if (realSource === realDest) {
+    return;
+  }
+
+  if (!fs.existsSync(entryFile)) {
+    console.error('Non-existent source file');
+
+    return;
+  }
```

The `fs.existsSync(entryFile)` check (line 43) is redundant because `fs.realpathSync(entryFile)` above would have thrown if the source didn't exist. If you keep this check it must follow the same error behavior (log then throw) rather than returning silently. Consider removing the redundant check or change the `return` to throwing an Error after logging.